### DISCUSSION TITLE
ctrk-k is the command for firefox's search

### DISCUSSION THREAD

**Discussion by sean-abbott** [community/community#15208] [Other Feature Feedback, Questions, & Ideas]
Which I still want available so I strongly dislike when websites override it.

**Comment by queenofcorgis** [comment]
Hi there @sean-abbott  👋🏼  Thanks for the feedback, we have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.

**Reply by bhrgunatha** [reply]
CTRL-K is a built-in browser shortcut. 

Please change the default to something that **doesn't hijack the browser's behaviour.**. It's incredibly annoying.       

The page you link does **not** let me change any shortcuts, it only lets me opt in or out of character keys: 

![image](https://user-images.githubusercontent.com/914363/164981533-0eabb049-f0cf-44f1-b5e6-194499bd6f63.png)

**Reply by Tenchumaru** [reply]
@liligalante, you marked this as an answer and perhaps it is for some but it is not for me or @bhrgunatha and likely others as well. Luckily, I disabled it when it was presented since I knew Ctrl+K was a browser shortcut. However, I see only the options @bhrgunatha sees. I see no way to change the keyboard shortcut for the command palette.

**Reply by dbitouze** [reply]
This issue is now fixed. Indeed, at the page https://github.com/settings/accessibility, you can modify the shortcut to trigger the Command Palette for the default search mode (here <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>p</kbd> instead of the default <kbd>Ctrl</kbd>+<kbd>k</kbd>):

![Capture d’écran_2022-09-14_15-52-06](https://user-images.githubusercontent.com/1032633/190173160-81c7b127-1272-46d7-8c3d-df535c802a9a.png)

**Reply by bhrgunatha** [reply]
Good for you! It isn't fixed for me though.     
[Here's the full accessibility page that I see](https://i.imgur.com/O5Bkqiz.png)

**Reply by dbitouze** [reply]
Quite strange! See [Customizing your GitHub Command Palette keyboard shortcuts](https://docs.github.com/en/get-started/using-github/github-command-palette#customizing-your-github-command-palette-keyboard-shortcuts).

**Reply by bhrgunatha** [reply]
Not strange, I'm just dumb.
I disabled the feature (because I use CTRL+K) extensively, then forgot it was disabled.   
After I re-enable it I can configure those settings now. :)

**Reply by Tenchumaru** [reply]
It took me forever to figure out that I had to select `Feature preview` from the user menu to enable this.
